examples/movement/physics_in_fixed_timestep.rs

```diff
@@ -41,8 +41,7 @@
 //! This is physically correct, but visually jarring. Imagine a player moving in a straight line, but depending on the frame rate,
 //! they may sometimes advance by a large amount and sometimes not at all. Visually, we want the player to move smoothly.
 //! This is why we need to separate the player's position in the physics simulation from the player's position in the visual representation.
-//! The visual representation can then be interpolated smoothly based on the last rendered position and
-//! the player's actual position in the physics simulation.
+//! The visual representation can then be interpolated smoothly based on the last and current actual player position in the physics simulation.
```

Comment:
```suggestion
//! The visual representation can then be interpolated smoothly based on the previous and current player position in the physics simulation.
```

Reply:
Good point, I'll fix the type names as well.

Reply:
I wasn't aware this referenced the type name, just the convention I use, sorry for trouble

Reply:
[`e92f15d` (#14223)](https://github.com/bevyengine/bevy/pull/14223/commits/e92f15d96687652dcc71ea8c175c57864cd96875)